omni-slider.js

```diff
@@ -200,6 +202,17 @@ Slider.prototype.init = function (options) {
   }
 };
 
+Slider.prototype._applyCallback_ = function (data, callback) {
+
```

Comment:
No, it's all good. We all have style- I don't know if this repo has a style. I'd ask @chrisrng otherwise don't worry about it.